src/apps/AppCenter/InstalledApps/InstalledApps.js

```diff
@@ -1,67 +1,77 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import PropTypes from 'prop-types'
-import { Viewport } from '@aragon/ui'
+import { useLayout, Card, CardLayout, GU } from '@aragon/ui'
 import { RepoType } from '../../../prop-types'
-import { ZoomableCards } from '../../../components/ZoomableCards'
-
-import AppsGrid from './AppsGrid'
 import AppContent from './AppContent'
 import AppCardContent from './AppCardContent'
 import RepoVersions from './RepoVersions'
```

Comment:
We should update this component with the designs:

<img width="287" alt="Screen Shot 2019-08-26 at 7 09 35 PM" src="https://user-images.githubusercontent.com/4166642/63708557-1012cc00-c835-11e9-9bfa-8027bdf748d2.png">